tests/large/provision/docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: "3.3"
 services:
   splunk:
     hostname: eventgensplunk
-    build: .
+    image: provision_splunk:latest
```

Comment:
Since docker-compose is not used, I suppose `docker-compose.yml` is not used any more?